docs/MANUAL.md

```diff
@@ -13,10 +13,12 @@ Under the `util` folder there is a command line tool named `lgptconvert.py` that
 
 ## Importing Samples
 
-picoTracker copies samples from the SD card into Flash memory for playing, this limits the amount of samples space to 1MB. It supports 8 or 16 Bit wav files, any sampling frequency, mono or stereo. 8bit samples are converted to 16bit at load time for compatibility with the engine (you can save space in storage but not in RAM, keep this in mind when choosing samples).
+picoTracker copies samples from the SD card into Flash memory for playing, this limits the amount of samples space to 1MB on pico based picoTracker or 16MB on a "portable model" picoTracker. It supports 8 or 16 Bit wav files, any sampling frequency, mono or stereo. 8bit samples are converted to 16bit at load time for compatibility with the engine (you can save space in storage but not in RAM, keep this in mind when choosing samples).
```

Comment:
We can use picoTracker Portable I think, for lack of better names. We can otherwise state that the entirety of the flash available can be used for samples minus the space used by the firmware, which is about 1MB, and that it will depend on the hardware used.

(not much preference about what we say, just an idea)